docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,237 @@
 ---
-title: Getting Started with Desktop Access
-description: Passwordless access to Windows systems for local users
+title: Configure access for local Windows users
+description: Configure passwordless access for local Windows users on Windows computers that aren't joined to an Active Directory domain.
 videoBanner: 9DyKQbg4ORc
 ---
 
-This guide will help you configure Teleport to provide secure, passwordless access
-to Windows desktops. This configuration does not require an Active Directory domain.
+This guide demonstrates how to configure Teleport to provide secure, passwordless access
+to Microsoft Windows desktops that aren't joined to an Active Directory domain.
```

Comment:
Is the description rendered on the page already? If it is, we probably don't need to say this twice.

Reply:
The description frontmatter is used by web crawlers, not rendered as text on the page (that's probably configurable but not typical).